dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesFile.java

```diff
@@ -51,6 +52,11 @@ public String getName() {
         // If the file was provisioned locally from an entry on Dockstore, we can remove the path to the temporary directory
         // from the file's absolute path to obtain a relative path.
         if (this.removablePrefix != null) {
+
+            if (!(Path.of(removablePrefix)).isAbsolute()) {
+                errorMessage(MessageFormat.format("Unable to calculate relative name for HTTP file attachment: {0}", super.getName()), CLIENT_ERROR);
```

Comment:
I'm still not clear what I would do if I saw this message. As an end-user, what did I do wrong, and how can I fix it?

Reply:
I don't think this will ever be surfaced to the user unless we internally use the class incorrectly, so it's more of a logical sanity check. That being said, if this error does appear, the user's best bet is to change the way they are specifying the attachment file.

Perhaps theres a better error type for this? It's more of an `INTERNAL_ERROR` (doesn't exist in the CLI) rather than a `CLIENT_ERROR`. I could also just throw an exception...